Engine/platform/base/agsplatformdriver.cpp

```diff
@@ -90,7 +90,10 @@ void AGSPlatformDriver::WriteStdOut(const char *fmt, ...) {
 }
 
 void AGSPlatformDriver::YieldCPU() {
-    std::this_thread::yield();
```

Comment:
This was interesting. The original timing method relies on this being a delay. If you use a proper yield then the cpu usage jumps up to 100% again.